terraform/environments/preview/variables.tf

```diff
@@ -1,34 +1,8 @@
 variable "aws_main_account_id" {}
 variable "aws_dev_account_id" {}
 
-variable "admin_user_ips" {
-  type = "list"
-}
-
-variable "dev_user_ips" {
-  type = "list"
-}
-
-variable "user_ips" {
-  type = "list"
-}
-
-variable "ssh_key_name" {}
-
-variable "g7_draft_documents_s3_url" {}
-variable "documents_s3_url" {}
-variable "agreements_s3_url" {}
-variable "communications_s3_url" {}
-variable "reports_s3_url" {}
-variable "submissions_s3_url" {}
```

Comment:
Aren't these used in the router nginx template? That's why I added `reports_s3_url` last week...